### PR TITLE
Jail Edit tooltip review

### PR DESCRIPTION
**Pull request by miker**
Updated the tooltips to current formatting practices
Updated tooltips for grammar and to reflect actaul usage

### CODE REVIEW

File: src/app/pages/jails/jail-edit/jail-edit.component.ts [hunk 1]
```diff
@@ -122,535 +131,644 @@ export class JailEditComponent implements OnInit {
...
+                 <b>A jail without a default route will not be \
+                 able to access the outside world without additional \
+                 configuration.'),
     },
     {
       type: 'input',
       name: 'notes',
       placeholder: T('Note'),
-      tooltip: T('Optional. Add any notes about the jail here.'),
+      tooltip: T('Store notes about the jail.'),
```
Comment (Mrt134) [inline src/app/pages/jails/jail-edit/jail-edit.component.ts]: Could just be a style thing, but "Store" doesn't feel right to me. Maybe "Save", "Add", or "Enter"?

File: src/app/pages/jails/jail-edit/jail-edit.component.ts [hunk 2]
```diff
@@ -122,535 +131,644 @@ export class JailEditComponent implements OnInit {
...
     },
     {
       type: 'checkbox',
       name: 'boot',
       placeholder: T('Auto-start'),
-      tooltip: T('Check to auto-start this jail at system boot time. Jails\
- are started and stopped based on the <b>priority</b> value.'),
-    }];
+      tooltip: T('Check to auto-start the jail at system boot time. \
```
Comment (Mrt134) [inline src/app/pages/jails/jail-edit/jail-edit.component.ts]: check -> set

File: src/app/pages/jails/jail-edit/jail-edit.component.ts [hunk 3]
```diff
@@ -662,32 +780,39 @@ export class JailEditComponent implements OnInit {
...
- <b>Example:</b> <i>/data example-dataset-name</i>'),
-    },];
+                 <b>Example:</b> <i>/data example-dataset-name</i>'),
+    },
+  ];
   public rctlConfig: FieldConfig[] = [
+// Spoke to Lola. Lines below starting here down to the "wallclock"
+// tooltip are commented out in jail-add.component, and are also
+// commented out of the HTML for the jail-edit page. Adding this comment
+// to clairify why these aren't reviewed currently.
```
Comment (Mrt134) [inline src/app/pages/jails/jail-edit/jail-edit.component.ts]: sp: clairify/clarify

File: src/app/pages/jails/jail-edit/jail-edit.component.ts (hunk 2 above)
Comment (miker) [reply src/app/pages/jails/jail-edit/jail-edit.component.ts]: fixed

File: src/app/pages/jails/jail-edit/jail-edit.component.ts [hunk 4]
```diff
@@ -122,535 +131,644 @@ export class JailEditComponent implements OnInit {
...
-      tooltip: T('This setting controls the availability of\
- IPv4 addresses. Possible values are <i>inherit</i> to allow\
- unrestricted access to all system addresses, <i>new</i> to\
- restrict addresses with <b>ip4_addr</b>, and <i>disable</i> to\
- stop the jail from using IPv4 entirely.'),
+      tooltip: T('Control the availability of IPv4 addresses. <br>\
+                 <b>Inherit</b>: Allow unrestricted access to all \
+                 system addresses. \
+                 <b>New</b> Restrict addresses with <b>ip4_addr</b>. \
+                 <b>Disable</b> Stop the jail from using IPv4 entirely.'),
```
[the remark this answers is not in the document]
Comment (miker) [reply src/app/pages/jails/jail-edit/jail-edit.component.ts]: fixed

**Comment by erinix** [comment]
please merge master to bring in the build fix

**Pull request merged**